notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>Maybe add:</p><p> </p><p>Observe that, in a vector space, tangent vectors are elements of the manifold and vice versa. A point in a vector space is a vector and a tangent vector, and conversely a tangent vector is a vector and a point in the vector space. In the implementation, we can see it in the <code>is_tangent()</code> method: to check that a vector is a tangent we just check that the vector belongs to the vector space. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>Is this consistent with what you are saying above?</p><p> </p><p>Observe that, in a <code>VectorSpace</code> manifold, the manifold is comprised of vectors -- and more specifically, tangent vectors. Thus, in the implementation, we can see that in the&nbsp;<code>is_tangent()</code>&nbsp;method: in order to check that a vector is a tangent to the manifold, we only need to check that the vector belongs to the vector space.</p><p> </p>